### PR TITLE
Add asset icons

### PR DESCRIPTION
**Pull request by carlfairclough**

### CODE REVIEW

File: burner-ui/src/components/BalanceItem/BalanceItem.tsx [hunk 1]
```diff
@@ -3,6 +3,39 @@ import { Asset } from '@burner-wallet/assets';
 import { Card, Box, Flex } from 'rimble-ui';
 import styled from 'styled-components';
 import Text from '../Text';
+import { Eth, Xdai, Dai } from '../TokenIcons';
+
+interface BalanceItemProps {
+  asset: Asset | string;
```
Comment (dmihal) [inline burner-ui/src/components/BalanceItem/BalanceItem.tsx]: Seems like some of these problems may be coming from accepting both of these as types, this should probably just be Asset right?

**Review by dmihal — CHANGES_REQUESTED** [review]

File: burner-ui/src/components/BalanceItem/BalanceItem.tsx (hunk 1 above)
Comment (carlfairclough) [reply burner-ui/src/components/BalanceItem/BalanceItem.tsx]: yeah you're right

Comment (carlfairclough) [reply burner-ui/src/components/BalanceItem/BalanceItem.tsx]: Updated 👍

**Comment by dmihal** [comment]
Thanks for the quick fix @carlfairclough, merging

**Pull request merged**